src/utils/sourceStyles.js

```diff
@@ -4,30 +4,24 @@ export const sourceStyles = {
     minHeight: '300px',
     maxWidth: '100%', // Fill parent before overflowing
     maxHeight: '100%', // Fill parent before overflowing
-    borderRadius: '5px',
-    overflow: 'auto',
-    cursor: 'text'
+    overflow: 'auto'
   },
   content: {
     padding: '20px',
     height: '100%',
-    fontSize: '15px'
   },
   inputArea: {
     display: 'inline-flex',
     width: '100%'
   },
   prompt: {
-    paddingTop: '3px'
+
   },
   input: {
     border: '0',
-    padding: '0 0 0 7px',
```

Comment:
Why were the changes in this block made? To me, it only seems they completely break the terminal.

Reply:
I would like to write those css in a classname-based css files.
This default style do not fit my usage somehow so I deleted it.